R/pkg/R/mllib.R

```diff
@@ -358,19 +369,11 @@ setMethod("summary", signature(object = "KMeansModel"),
                    cluster = cluster, is.loaded = is.loaded))
           })
 
-#' Predicted values based on model
-#'
-#' Makes predictions from a k-means model or a model produced by spark.kmeans().
+#  Predicted values based on a k-means model
 #'
-#' @param object A fitted k-means model
-#' @rdname predict
```

Comment:
or maybe we can just remove it from R docs? anyway predict should be a separate generic method across all ml algorithmns